contracts/express/ExpressService.sol

```diff
@@ -34,6 +36,42 @@ contract MockGMPExpressService is AxelarExecutable, IGMPExpressService {
         _;
     }
 
+    function isExpressProxy(address proxyAddress) public view returns (bool) {
+        return proxyDeployer.isExpressProxy(proxyAddress);
+    }
+
+    function deployedProxyAddress(bytes32 salt, address sender) external view returns (address) {
+        return proxyDeployer.deployedProxyAddress(salt, sender, address(this));
+    }
+
+    function deployExpressProxy(
+        bytes32 salt,
+        address implementationAddress,
+        address owner,
+        bytes calldata setupParams
+    ) external returns (address) {
+        bytes32 deploySalt = keccak256(abi.encode(msg.sender, salt));
+        return _deployExpressProxy(deploySalt, implementationAddress, owner, setupParams);
+    }
+
+    function _deployExpressProxy(
+        bytes32 deploySalt,
+        address implementationAddress,
+        address owner,
+        bytes memory setupParams
+    ) internal returns (address deployedAddress) {
+        (, bytes memory data) = address(proxyDeployer).delegatecall(
```

Comment:
@deanamiel could you please move this code right into `deployExpressProxy` since we don't reuse `_deployExpressProxy` anywhere else.